src/app/canvastable/canvastable.ts

```diff
@@ -512,7 +515,7 @@ export class CanvasTableComponent implements AfterViewInit, DoCheck {
             (this.hoverRowIndex - this.topindex) * this.rowheight,
             colStartX - this.horizScroll + this.colpaddingleft,
             this.columns[colIndex].width - this.colpaddingright - this.colpaddingleft,
-            this.rowheight, tooltipText);
+            this.rowheight, tooltipText as string);
```

Comment:
I think it's guaranteed to be a string by now (thanks to L500), so this is just to make Typescript happy?

Reply:
Yes typescript was not convinced that this had to be a string at this point ( it seems normally quite intelligent on these matters, but not in this case ).